src/Components/Web.JS/src/Platform/Circuits/DefaultReconnectDisplay.ts

```diff
@@ -72,11 +72,11 @@ export class DefaultReconnectDisplay implements ReconnectDisplay {
         // - true to mean success
         // - false to mean we reached the server, but it rejected the connection (e.g., unknown circuit ID)
         // - exception to mean we didn't reach the server (this can be sync or async)
-        const successful = await Blazor.reconnect!();
+        const successful = await Blazor.reconnect?.();
```

Comment:
This seems like a functional change. It's making the code tolerate `reconnect` being not set, whereas previously we were (in effect) asserting that it was set. Unless there's a scenario where being unset should be tolerated, could we revert this? Thanks!

Reply:
Reverted!